src/init/features/emulators.ts

```diff
@@ -76,7 +76,7 @@ export async function doSetup(setup: any, config: any) {
       if (ui.enabled) {
         await prompt(ui, [
           {
-            type: "input",
+            type: "number",
```

Comment:
Does empty string (any port) still work here?

Reply:
Good question and yes, I was able to produce this:
```json
{
  "emulators": {
    "firestore": {
      "port": 8040
    },
    "ui": {
      "enabled": true
    }
  }
}
```